src/app/new-char/page.tsx

```diff
@@ -1,10 +1,10 @@
 'use client';
 
-import { Form, Wrapper } from '@/app/style';
 import { Button } from '@/app/player/[characterId]/[state]/style';
 import React, { useState } from 'react';
 import useRealtimeState from '@/app/hooks/useRealtimeState';
 import type { pjData } from '@/@types/pjData';
+import { Form, Input, Label, Wrapper } from '@/app/login/style';
```

Comment:
Extrair esses componentes da página de _login_ e levar para a pasta `src/components` os componentes utilizados por ambas as páginas, para uso comum sem forçar a dependência entre as páginas